packages/website/src/pages/docs/Guides/DistributionCreation.mdx

```diff
@@ -280,22 +202,22 @@ Creates a [beta distribution](https://en.wikipedia.org/wiki/Beta_distribution) w
   ]}
 >
   <Tab>
-    <SquiggleEditor defaultCode="beta(10,20)" />
+    <SquiggleEditor defaultCode="Sym.beta(10,20)" />
```

Comment:
Yea, we should add more information about ``Sym`` somewhere, I'm not sure what's best.

I used it here because it looks a lot better - in the sense that it would give people a better idea of what these distributions are.